cmd/config/api/api.go

```diff
@@ -172,7 +178,13 @@ func LookupConfig(kvs config.KVS) (cfg Config, err error) {
 	if err != nil {
 		return cfg, err
 	}
-
+	replicationWorkers, err := strconv.Atoi(env.Get(EnvAPIReplicationWorkers, kvs.Get(apiReplicationWorkers)))
+	if err != nil {
+		return cfg, err
+	}
+	if replicationWorkers == 0 {
+		replicationWorkers = 1 // set minimum replicationWorkers to 1
```

Comment:
We can error out `if replicationWorkers == 0`

Reply:
done